src/mon/MonClient.cc

```diff
@@ -264,13 +264,41 @@ bool MonClient::ms_dispatch(Message *m)
   }
 
   Mutex::Locker lock(monc_lock);
+  ldout(cct, 10) << "[ms_dispatch] : incoming msg [" << *m << "] from addr [" 
```

Comment:
I actually like the [] format, but nothing else uses it.  Most things use << __func__ << ": ... to indicate which method it is.